homeassistant/components/wled/light.py

```diff
@@ -166,6 +170,22 @@ def rgbw_color(self) -> tuple[int, int, int, int] | None:
             return None
         return cast(tuple[int, int, int, int], color.primary)
 
+    @property
+    def color_temp_kelvin(self) -> int | None:
+        """Return the CT color value in K."""
+        cct = self.coordinator.data.state.segments[self._segment].cct
+        return kelvinTo255Reverse(cct, COLOR_TEMP_K_MIN, COLOR_TEMP_K_MAX)
+
+    @property
+    def min_color_temp_kelvin(self) -> int:
+        """Return the warmest K."""
+        return COLOR_TEMP_K_MIN
```

Comment:
These are static, meaning we can just set the shorthand attr values (`_attr_min_color_temp_kelvin`).